### PR TITLE
Reformatted batch #3 (see comment) using flake8.

### PR DESCRIPTION
**Pull request by jakleh**
Files reformatted:

-dataprofiler/labelers/char_load_tf_model.py
-dataprofiler/reports/utils.py
-dataprofiler/reports/graphs.py
-dataprofiler/profilers/data_labeler_column_profile.py
-dataprofiler/profilers/__init__.py
-dataprofiler/labelers/labeler_utils.py
-dataprofiler/data_readers/__init__.py
-resources/__init__.py
-dataprofiler/data_readers/filepath_or_buffer.py
-dataprofiler/profilers/helpers/report_helpers.py

### CODE REVIEW

File: dataprofiler/reports/graphs.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python3
+"""Contains functions for generating graph data report."""
+# !/usr/bin/env python3
```
Comment (taylorfturner) [inline dataprofiler/reports/graphs.py:2]: we need this? `!/user/bin/env python3`?

Comment (jakleh) [reply dataprofiler/reports/graphs.py:2]: It was already there.

Comment (JGSweets) [reply dataprofiler/reports/graphs.py:2]: fine to leave imo

**Review by taylorfturner — APPROVED** [review]

File: dataprofiler/reports/graphs.py (hunk 1 above)
Comment (taylorfturner) [reply dataprofiler/reports/graphs.py:2]: for reference, https://predictivehacks.com/?all-tips=a-shebang-line-usr-bin-python3#:~:text=Its%20purpose%20is%20to%20define,script%20as%20python3%20file.py%20

**Review by JGSweets — APPROVED** [review]

**Pull request merged**